ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -106,22 +171,20 @@ The criterion to be compared with is defined in the configuration.
 - *Value*   
     Enter the value to be compared with the input value. The value must be valid JSON and is type strict, for example a string must be specified as a string (**\"example\"**), an integer as an integer (**123**), and so on.
 
-[comment]: <> (Ticket ICBPM-197 in Arbeit: Weitere operatoren geplant: <=, <, >=, >)
-
 
 
 ## Start subprocess
 
 ![Start subprocess](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/StartSubprocess.png "[Start subprocess]")
 
-The *Start subprocess* core action is used to start a different subprocess within the currently running process.       
-The data runs via the *p* input port into the workflow action and is output via the *p* output port. However, the action does not wait until the subprocess has been finished, but outputs the data after having started the subprocess. The input data is also the data put in the start place of the subprocess.   
-The subprocess to be started is defined in the configuration.
+The *Start subprocess* core action is used to start a different sub process within the currently running process.       
```

Comment:
Ich finde es separat (sub process) schwieriger zu lesen, ehrlich gesagt. Merriam Webster sagt zusammen oder mit Bindestrich.
https://www.merriam-webster.com/dictionary/subprocess
In unserer Style Guide habe wir ähnliche Termini mit Bindestrich: sub-tab, sub-procedure... 
https://actindo.atlassian.net/wiki/spaces/DOC/pages/110886966/Terminology+for+Terminologue
Also lieber mit Bindestrich?